frontend/catalyst/utils/jax_cpu_lapack_kernels/lapack_kernels_using_lapack.cc

```diff
@@ -0,0 +1,152 @@
+/* Copyright 2021 The JAX Authors.
```

Comment:
Minor: could we use cpp extension instead of cc?

Reply:
Fixed.